src/app/shared/business/asset-detail-dialog/asset-property-grid-group-builder.ts

```diff
@@ -101,15 +117,15 @@ export class AssetPropertyGridGroupBuilder {
 
       fields.push({
         icon: 'api',
-        label: 'HTTP Data Source Parameterization',
+        label: this.translateService.instant('component_library.http_param'),
```

Comment:
remove `component_library` prefix, maybe move to "data_source" or "data_address" as parent term for both data source and data sink